flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/context/SessionContextTest.java

```diff
@@ -129,17 +132,20 @@ public void testSetAndResetArbitraryKey() {
     // --------------------------------------------------------------------------------------------
 
     private SessionContext createSessionContext() {
-        Configuration flinkConfig = new Configuration();
-        flinkConfig.set(OBJECT_REUSE, true);
-        flinkConfig.set(MAX_PARALLELISM, 16);
+        Map<String, String> flinkConfig = new HashMap<>();
```

Comment:
why modify this?

Reply:
Because I have modified the `SessionContext#create`, so this method for test needs to be modified synchronously.